react/stories/channel.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { storiesOf } from '@storybook/react';
+import PropTypes from 'prop-types';
+import channelMetadataFragment from 'react/components/ChannelMetadata/fragments/channelMetadata';
+import ChannelBreadcrumb from 'react/components/ChannelMetadata/components/ChannelBreadcrumb';
+import ChannelMetadataInfo from 'react/components/ChannelMetadata/components/ChannelMetadataInfo';
+
+class MockChannel extends Component {
```

Comment:
Again, probably a nicer way to do this, just offering an example.

Reply:
What's with the naming here? Be consistent with classes and their filenames.

Reply:
This is not the focus of the file, just a helper component to fake the Apollo fetches.

Reply:
Then I'd separate it out. One class per-file. I thought there was an eslint rule for this?

Reply:
Though generally this is sorta weird. Why not use the actual Channel component?

Reply:
I don't want the entire `ChannelMetadata` component, just want a query to fetch a mocked channel.

Reply:
Mergeit